CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.0.8
+
+- ### Features
+
+  - **Support a variety of errors thrown from `getAssetFromKV` - [victoriabernard92], [issue/59] [pull/64]**
+
+    Support for throwing specific errors to allow the handler of `getAssetFromKV` to catch and differentiate them.
+    For example, a 404 `NotFoundError` error implies nothing went wrong, the asset just didn't exist while
+    a 500 `InternalError` means an expected variable was undefined.
+
+    [victoriabernard92]: https://github.com/victoriabernard92
+    [issue/44]: https://github.com/cloudflare/kv-asset-handler/issues/44
+    [issue/59]: https://github.com/cloudflare/kv-asset-handler/issues/59
+    [pull/47]: https://github.com/cloudflare/kv-asset-handler/pull/47
+
+- ### Fixes
+
+  - **Range Issue with Safari and videos - [victoriabernard92], [issue/60] [pull/66]**
+
+    Support passing in a Request instead of simply a URL string into the cache as the cache key to prevent
```

Comment:
```suggestion
 Previously, if you wanted to serve a video from Workers KV using `kv-asset-handler`, it would be broken on Safari due to its requirement that all videos support the [`Content-Range` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Range). Cloudflare already has a feature that will handle these headers automatically, we just needed to take advantage of it by passing in a `Request` object to the [Cache API](https://developers.cloudflare.com/workers/reference/apis/cache/) rather than a URL string.
```